editor/import/import_defaults_editor.cpp

```diff
@@ -195,11 +327,43 @@ void ImportDefaultsEditor::clear() {
 			_update_importer();
 		}
 	}
+	// Only triggers if anything is selected
```

Comment:
```suggestion
	// Only triggers if anything is selected.
```
Same in other comments.